src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Serialization/RazorProjectInfoDeserializer.cs

```diff
@@ -1,8 +1,10 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT license. See License.txt in the project root for license information.
 
+using System;
 using System.IO;
 using Microsoft.AspNetCore.Razor.ProjectSystem;
+using StreamJsonRpc;
```

Comment:
Why is this using directive needed?

Reply:
It's not, possibly got added by one of the code fixes automagically. Removed.